### ISSUE TITLE
Database help

### ISSUE THREAD

**Issue by dimon0230ru** [Dantevg/WebStats#10]
can i convert offline uuid from uuid column to nickname in MYCUSTOMTABLE table? I configured it like this but nothing worked

database:
  hostname: localhost
  username: DATABASE USERNAME
  password: DATABASE PASSWORD
  config:
    - database: DATABASE NAME
      table: MYCUSTOMTABLE
      convert: # Some examples, refer to documentation for explanations
        - [uuid, column]
 
![image](https://user-images.githubusercontent.com/55211830/136770829-5d025345-0847-47c8-93ef-95d93795fbb4.png)

**Comment by dimon0230ru** [comment]
no 
![image](https://user-images.githubusercontent.com/55211830/136773693-844a9ba8-0d18-4d67-8539-b3af7db46de9.png)

**Comment by dimon0230ru** [comment]
Should it create a player column in this table and write a nickname there?

**Comment by dimon0230ru** [comment]
It will be wonderful. Almost all normal economy plugins are made for uuid, but my site only accepts a nickname to synchronize currency between servers and site

**Comment by Dantevg** [comment]
Ok, some questions: 
- to clarify, your server is in offline mode? 
- with nickname, do you mean ingame name or really the nickname? (might be an EssentialsX thing? Often indicated with `~`)
- so you essentially need a database table with UUID/nickname combinations so your website can look up the nickname for a given uuid?

**Comment by dimon0230ru** [comment]
yes, the server is offline. I have CMI instead of EssentialX. Like this% player_name%, game nickname. The site can only connect to a table with columns nickname and balance, the name of the columns can be changed. And the economy plugin to uuid and balance (pocket - column). It is necessary that in one table there should be uuid and username and pocket (balance). 
Economy plugin called MZP-Economy and MZP-Core API
